Lib/test/test_dynamic.py

```diff
@@ -134,7 +134,8 @@ def test_eval_gives_lambda_custom_globals(self):
 
         self.assertEqual(foo(), 7)
 
-    @unittest.skipIf(is_wasi, "stack depth too shallow in WASI")
+    @unittest.skipIf(is_wasi and Py_DEBUG,
+                     "stack depth too shallow in WASI debug build")
     def test_load_global_specialization_failure_keeps_oparg(self):
```

Comment:
Now that `Py_C_RECURSION_LIMIT` under `Py_DEBUG` is the same for WASI as everything else, why do we need to skip this test at all?  I suspect that you found the test to still fail on WASI under `Py_DEBUG`, but it is unclear why WASI would fail here where other platforms don't.

Reply:
I was just too lazy to test. In short, the test just works in debug mode as well :-)